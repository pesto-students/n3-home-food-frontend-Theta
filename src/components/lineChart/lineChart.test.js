import * as React from "react";
import Enzyme, { shallow } from "enzyme";
import EnzymeAdapter from "enzyme-adapter-react-16";
import LineChart from "./lineChart";

Enzyme.configure({ adapter: new EnzymeAdapter() });

describe("Line Chart component", () => {
  let wrapper;
  test("render without error", () => {
    wrapper = shallow(<LineChart/>);
    expect(wrapper.length).toBe(1);
  });

  test('should render correctly in "debug" mode', () => {
    const component = shallow(<LineChart />);

    expect(component).toMatchSnapshot();
  });
});