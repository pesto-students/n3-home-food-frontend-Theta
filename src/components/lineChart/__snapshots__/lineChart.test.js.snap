// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Line Chart component should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
